rust/main/ethers-prometheus/src/middleware/mod.rs

```diff
@@ -335,10 +335,9 @@ impl<M: Middleware> Middleware for PrometheusMiddleware<M> {
                     .inc_by((Instant::now() - start).as_secs_f64());
             }
         }
-
         Ok(result?)
     }
-
+    #[allow(clippy::redundant_closure)] // TODO: `rustc` 1.80.1 clippy issue
```

Comment:
nit: probably want to leave the newline above this line

Reply:
Added ✅

Reply:
Removed the line again due to it being a `cargo fmt` issue 🤷